vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -67,19 +88,48 @@ int main(int argc, char** argv)
         return l_rc;
     }
 
-    if (*l_hardwareFlag && !std::filesystem::exists(l_vpdPath))
-    {
-        std::cerr << "Given EEPROM file path doesn't exist : " + l_vpdPath
-                  << std::endl;
-        return l_rc;
-    }
-
-    (void)l_fileOption;
-
     if (*l_readFlag)
     {
         // TODO: call read keyword implementation from here.
     }
+    else if (l_writeFlag->count() > 0)
```

Comment:
if l_writeFlag->count() == 1

Reply:
if you keep "==1", then we need to check "==0" and ">1" as well, 
which leads more check and log.
Problem is only when user doesn't provide one, providing option multiple times we can ignore and give the result.

This behaviour is similar how linux command works.